### PR TITLE
feat: Add --raw flag to send-envelope

### LINKED ISSUE #1458: Add --raw flag to send-envelope command
This should send the the envelope without attempting to parse it.

Suggestion from @kamilogorek:

> change sentry-cli send-envelope to accept --raw flag, that instead of doing
> ```
> with_sentry_client(dsn, |c| c.send_envelope(Envelope::from_path(p))
> ```
> will do
> ```
> // pseudo code
> if matches.is_present("raw") {
>   with_sentry_client(dsn, |c| c.send_request(read_file(p)))`
> }
> ```

### PR DESCRIPTION
**Pull request by loewenheim**
This adds a flag `--raw` to the `send-envelope` command that will cause the envelope to be sent without being parsed first.

Since this uses the new `Envelope::from_path_raw` function, the `sentry-rust` dependency is updated to 0.29.3.

Fixes #1458.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 "You know what they say. Fool me once, strike one, but fool me twice... strike three." — Michael Scott
 
+## Unreleased
+
+### Various fixes & improvements
+
+- feat: Added the `--raw` flag to `send-envelope` to allow sending envelopes without parsing
```
Comment (kamilogorek) [inline CHANGELOG.md:9]: Afaik we never wrap on changelog entries

File: src/commands/send_envelope.rs [hunk 2]
```diff
@@ -26,6 +26,12 @@ pub fn make_command(command: Command) -> Command {
                 .required(true)
                 .help("The path or glob to the file(s) in envelope format to send as envelope(s)."),
         )
+        .arg(
+            Arg::new("raw")
+                .long("raw")
+                .action(ArgAction::SetTrue)
```
Comment (kamilogorek) [inline src/commands/send_envelope.rs:32]: This is redundant in the version of clap that we use here.

Comment (kamilogorek) [inline src/commands/send_envelope.rs:32]:
```suggestion
```

File: src/commands/send_envelope.rs [hunk 3]
```diff
@@ -1,7 +1,7 @@
 use std::path::PathBuf;
 
 use anyhow::Result;
-use clap::{Arg, ArgMatches, Command};
+use clap::{Arg, ArgAction, ArgMatches, Command};
```
Comment (kamilogorek) [inline src/commands/send_envelope.rs:4]:
```suggestion
use clap::{Arg, ArgMatches, Command};
```

**Review by kamilogorek — CHANGES_REQUESTED** [review]

File: src/commands/send_envelope.rs (hunk 2 above)
Comment (loewenheim) [reply src/commands/send_envelope.rs:32]: That does not appear to work. Note that `matches.get_flag("raw")` is used later to get the value, and AFAICT that requires `ArgAction::SetTrue` or `ArgAction::SetFalse`.

Comment (kamilogorek) [reply src/commands/send_envelope.rs:32]: That's what I changed it to :) I just prefer being consistent

**Review by kamilogorek — APPROVED** [review]

**Pull request merged**